### DISCUSSION TITLE
Klaviyo_Reclaim

### DISCUSSION THREAD

**Discussion by eneskirca** [breezefront/community#99] [3. Integrations]
I am writing to inquire about the integration of Klaviyo Reclaim, an extension for email marketing, with your platform.

As a business owner, email marketing is an essential part of our customer acquisition and retention strategy. We recently came across Klaviyo Reclaim and were impressed with its ability to automate the email recovery process for abandoned carts and lost sales. We believe this extension could greatly benefit our business and help us increase revenue.

In order to take advantage of this opportunity, we need to ensure that Klaviyo Reclaim can be easily integrated with your platform. We would appreciate any information or assistance you can provide regarding this matter.

Thank you for your attention to this request, and I look forward to hearing back from you soon.

**Comment by eneskirca** [comment]
Klaviyo is one of the most popular apps for the e-commerce industry 
Here is GitHub link for the extension 
https://github.com/klaviyo/magento2-klaviyo

Do you have any plans to integrate it?

**Comment by jorgb90** [comment]
Yes I agree! Should be intergrated!

**Comment by dimdimich** [comment]
Please test integration from https://github.com/breezefront/module-breeze-klaviyo-reclaim
